### PR TITLE
Adding in setup.py

### PR DESCRIPTION
**Pull request by mortonjt**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    Programming Language :: Python :: 3
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS :: MacOS X
+"""
+classifiers = [s.strip() for s in classes.split('\n') if s]
+
+setup(name='americangut',
+      version=version,
+      description="American gut analyses",
```
Comment (wasade) [inline setup.py]: gut -> Gut

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      long_description="American Gut open-access data "
+                       "and IPython notebooks",
```
Comment (wasade) [inline setup.py]: notebooks -> Notebooks

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      author="American Gut development team",
+      author_email="jamietmorton@gmail.com",
```
Comment (wasade) [inline setup.py]: it would be more appropriate to list either me, Justine, Embriette or Rob

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      maintainer="American Gut development team",
+      maintainer_email="jamietmorton@gmail.com",
```
Comment (wasade) [inline setup.py]: Same here unless you'd like to be the maintainer :)

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      url='http://github.com/biocore/American-Gut/',
```
Comment (wasade) [inline setup.py]: http -> https

**Comment by wasade** [comment]
Few comments, :+1: otherwise once tests pass

File: setup.py (hunk 3 above)
Comment (wasade) [inline setup.py]: ...actually, Just use "American Gut development team" and the American Gut help account email.

**Comment by mortonjt** [comment]
:boom: tests passing

**Comment by squirrelo** [comment]
Already merged now, but did we really need a setup.py given that this is primarily a collection of notebooks and data? Would a pip requirements.txt and/or the conda equivalent have been more appropriate?

**Comment by wasade** [comment]
It makes deployment easier

**Comment by squirrelo** [comment]
eh, 
```bash
pip install .
```
vs 
```bash
pip install -r requirements.txt
echo $(pwd)/scripts >  .bash_profile
```
The second has an extra line, but also doesn't treat the repo as a full installable package. I guess the question for me is, should this be treated as a full installable package or a collection of notebooks?

**Comment by wasade** [comment]
What's the downside to it being installable?

**Pull request merged**